db/migrate/20170614135258_create_split_pdf_logs.rb

```diff
@@ -0,0 +1,22 @@
+class CreateSplitPdfLogs < ActiveRecord::Migration
+  def change
+    create_table :split_pdf_logs do |t|
+      t.string :split_pdf_console_log
+      t.string :error_description
+      t.string :uploaded_filetype, null: false
+      t.string :uploaded_filename, null: false
+      t.string :user, null: false
+      t.string :host, null: false
+      t.integer :number_of_new_files_in_raw_dir, null: false
```

Comment:
You definitely need to make these column names shorter. Also, for these directories it's more useful to know the number of groups rather than the number of pages.

Don't know what "raw" and "error" are, but I doubt both of them are necessary.